gammapy/time/exptest.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+
+__all__ = [
+    'exptest_for_run',
+]
+
+def exptest_for_run(time_delta=[]):
```

Comment:
Remove the default argument of an empty list here, i.e. write:
```python
def exptest_for_run(time_delta):
```
(because it doesn't make sense to call this function without passing any inputs.